bin/build-asset-cdn-json.php

```diff
@@ -9,7 +9,7 @@
 foreach ( $regex as $file => $value ) {
 	$file = str_replace( $path, '', $file );
 	$directory = substr( $file, 0, strpos( $file, '/' ) );
-	if ( in_array( $directory, array( 'node_modules', 'tests' ) ) ) {
+	if ( in_array( $directory, array( 'node_modules', 'tests', 'extensions' ) ) ) {
```

Comment:
Is this looking for `./extensions` or `./**/extensions`?

Reply:
I think we need to make sure it is `./extensions` right?

Reply:
Correct, we don't want to ignore `./something/extensions`

Reply:
tested, works as expected, that list involves just top level paths

Reply:
We're comparing the ignore list against the repo root. I've attempted to clarify things in 1915e8ba6f041b88d29e45819da9763d17a5de7e